backend/tests/core/ServiceHelper.cs

```diff
@@ -115,7 +115,7 @@ public static class ServiceHelper
         /// <param name="helper"></param>
         /// <param name="args"></param>
         /// <returns></returns>
-        public static T CreateService<T>(this TestHelper helper, params object[] args) where T : IService
+        public static T CreateService<T>(this TestHelper helper, params object[] args) where T : IRepository
```

Comment:
are these renames part of a different pr too?

Reply:
Ideally, we would have the business/features changes on a different pr than the tech enhancements to keep changes logical and easier to comb though

Reply:
That is the case, When that PR is merged these renames will magically disappear, just couldn't wait for Alejandro's PR any longer.

Reply:
Pr != commit

Reply:
Ah I see. Well the PR shows the difference between the state of the branch to be merged and the set of commits on it.